src/main/starlark/builtins_bzl/common/objc/compilation_support.bzl

```diff
@@ -28,6 +28,9 @@ load(":common/xcode/providers.bzl", "XcodeVersionInfo")
 
 objc_internal = _builtins.internal.objc_internal
 
+def _join_path(*path):
+    return "/".join([p for p in path if p != ""])
```

Comment:
https://github.com/bazelbuild/bazel/blob/48fe8619bf4afbd9d00f2d0ce32b27a950e09767/src/main/starlark/builtins_bzl/common/paths.bzl#L73 is a similar helper that already exists.

Reply:
thanks!